cupy/fft/_callback.pyx

```diff
@@ -89,7 +89,7 @@ cdef inline void _set_vars() except*:
         if not os.path.isfile(_nvprune):
             _nvprune = None
 
-    _build_ver = str(CUPY_CUDA_VERSION)
+    _build_ver = str(runtime.runtimeGetVersion())
```

Comment:
Thanks for the ping. I am wondering if we could remove `_build_ver` since we also record the cuFFT (runtime) version as part of the key. Perhaps I can explore it later...